### PR TITLE
Feature/IphoneX Support

### PR DESCRIPTION
**Pull request by ChildishDanbino**
Added detectIphoneX function to set isIphoneX variable to true or false based on IphoneX properties. 

README was updated to reflect changes and also added ES6 Module import example in ReadME.

Tested using simulator. Code is based on https://github.com/ptelad/react-native-iphone-x-helper

### CODE REVIEW

**Comment by ChildishDanbino** [comment]
@m0ngr31 - Any issues with this PR? Would be willing to make changes if it would get it approved. Let me know.

**Comment by m0ngr31** [comment]
Sorry, I haven't had a good chance to look it over yet (just had a baby!)

**Comment by ChildishDanbino** [comment]
Ahhh congrats! No rush on it but let me know if you need me to make any changes.

File: README.md [hunk 1]
```diff
@@ -11,7 +11,11 @@ Install
 Usage
 ------------------------
 
-Import into a project file: `const Device = require('react-native-device-detection');`
+Import into a project file: 
+
+Require Import: `const Device = require('react-native-device-detection');`
+
+ES6 Modules: `import { isAndroid } from 'react-native-device-detection`;
```
Comment (m0ngr31) [inline README.md:18]: Does this actually work?

Comment (ChildishDanbino) [reply README.md:18]: @m0ngr31 - Indeed it does. Here is an example from our Production Application. I copied and pasted from our Main.js

```import { isAndroid, isTablet } from 'react-native-device-detection';```

Comment (m0ngr31) [reply README.md:18]: Awesome

**Review by m0ngr31 — APPROVED** [review]

**Pull request merged**